src/DIRAC/Core/Utilities/Graphs/GraphUtilities.py

```diff
@@ -75,13 +67,19 @@ def convert_to_datetime(dstring):
             results = dstring
         else:
             results = eval(str(dstring), {"__builtins__": None, "time": time, "math": math}, {})
-        if isinstance(results, six.integer_types + (float,)):
+        if isinstance(
+            results,
+            (
+                int,
+                float,
+            ),
+        ):
```

Comment:
The trailing comma is significant and cause black to wrap the lines, I think this is clearer:
```suggestion
        if isinstance(results, (int, float)):
```